webui/react/src/ioTypes.ts

```diff
@@ -116,7 +116,7 @@ const commandStates: Record<string, null> = Object.values(CommandState)
 const commandStatesIoType = io.keyof(commandStates);
 
 export const ioGenericCommand = io.type({
-  addresses: io.union([ io.array(ioCommandAddress), io.null ]),
+  addresses: io.union([ io.array(ioCommandAddress), io.null, io.undefined ]),
```

Comment:
both `null` and `undefined` can happen here, crazy!

Reply:
ditto